packages/beacon-node/src/util/sszBytes.ts

```diff
@@ -6,6 +6,17 @@ import {BYTES_PER_FIELD_ELEMENT, FIELD_ELEMENTS_PER_BLOB} from "@lodestar/params
 export type BlockRootHex = RootHex;
 export type AttDataBase64 = string;
 
+// class BeaconState(Container):
+//   genesis_time: uint64              - 8
+//   genesis_validators_root: bytes32  - 32
+//   slot: uint64
+//   ...
+const STATE_SLOT_OFFSET = 40;
+
+export function getSlotFromBeaconStateSerialized(data: Uint8Array): Slot {
```

Comment:
can we use existing `getStateSlotFromBytes`?

https://github.com/ChainSafe/lodestar/blob/c6369ee5fe2c41f922e36afaedd7db4d8a69b15f/packages/beacon-node/src/util/multifork.ts#L44